scripts/index.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs')
+const path = require('path')
+//CO2e -> kilowattHours * emissions[region];
+function updateCO2eData() {
+    const data = fs.readFileSync(
+        path.resolve(__dirname, `../packages/client/stub-server/mockData.json`),
+        'utf8',
+    );
+    const mockData = JSON.parse(data);
+
+    mockData.footprint.forEach(({ serviceEstimates }) => {
+        serviceEstimates.forEach((data) => {
```

Comment:
rather than 'data', consider 'serviceEstimate' to avoid confusion with the other data variable in scope defined earlier